test/parallel/test-crypto-random.js

```diff
@@ -37,6 +37,17 @@ process.setMaxListeners(256);
   });
 });
 
+// crypto.addEntropy takes nothing or an ArrayBuffer
+[-1,
+ undefined,
+ null,
+ false,
+ true,
+ {}, [], [1]
+].forEach(function(value) {
+  assert.throws(function() { crypto.addEntropy(value); });
+});
+
```

Comment:
Is there any reason that all tests do not change random states and always throw 'Not a buffer'?